content/en/docs/components/training/_index.md

```diff
@@ -1,5 +1,5 @@
 +++
-title = "Training Operators"
+title = "Frameworks for Training"
```

Comment:
We made this change recently in the opposite direction. I believe that `Training operators` is more accurate.

Reply:
I'll fix it.